configs/ci/permissions/config-manager.json

```
@@ -1,4 +1,12 @@
 {
+    "activation_keys": [
+        {
+            "verb": "read"
+        },
+        {
+            "verb": "write"
+        }
```

Even though it's not able to be used for custom role creation, we should define the global `config-manager:activation_keys:*` permission in the permissions files since they're the source of truth for available permissions, given it's being used in a role.
```suggestion
        },
        {
            "verb": "*"
        }
```